modules/apps/frontend-icons/github-icons/package.json

```diff
@@ -0,0 +1,9 @@
+{
+	"main": "images/spritemap.svg",
```

Comment:
:thinking: Not sure if this is the way to go... If this is something we will be able to use from OSGi I'd rather move it to the BND instead, because we normally use the `package.json` for frontend-only things, and assume there's some frontend build tool involved.

As a similar example, I began the `<liferay-frontend:stylesheet>` feature relying on the bundler and the AMD loader and then came to the conclusion that moving everything to OSGi (which implies server-side execution) was far better.

I don't know if this will be the case, but if we are going to need to refer to icons from JSPs or Java files, this should probably be moved to BND.

> Note that I'm not saying that we can't do it in `package.json` (you can access this file from OSGi too, so there's no problem in putting things here), it's just that I feel we need to be sure which world (OSGi vs browser) things belong to. And for things available in both worlds, well, then we need to decide what fits best, I guess.

Reply:
> Not sure if this is the way to go... If this is something we will be able to use from OSGi I'd rather move it to the BND instead, because we normally use the package.json for frontend-only things, and assume there's some frontend build tool involved.

You are right, this doesn't make sense to be a package.json thing. We should make the package.json optional. The reason I thought it would be helpful is for custom build process via node or something.


> I don't know if this will be the case, but if we are going to need to refer to icons from JSPs or Java files, this should probably be moved to BND.

any suggestions on where to learn more about adding stuff to the bnd? I feel like an intern all over again jumping into some of this stuff 😂